Jenkinsfile

```diff
@@ -112,28 +112,28 @@ pipeline {
                 }
             }
         }
-	       stage('SonarQube Analysis') {
-            steps {
-            git branch: 'master', credentialsId: 'GITHUB_AB2D_JENKINS_PAT', url: env.GIT_URL
-            git branch: env.BRANCH_NAME, credentialsId: 'GITHUB_AB2D_JENKINS_PAT', url: env.GIT_URL
-                // Automatically saves the an id for the SonarQube build
-                withSonarQubeEnv('CMSSonar') {
-                    sh '''mvn sonar:sonar -Dsonar.projectKey=ab2d-project -DskipTests'''
-                }
-            }
-        }
-
-	  //New Way in declarative pipeline
-        stage("Quality Gate") {
-           options {
-                timeout(time: 10, unit: 'MINUTES')
-            }
-            steps {
-                // Parameter indicates whether to set pipeline to UNSTABLE if Quality Gate fails
-                // true = set pipeline to UNSTABLE, false = don't
-                waitForQualityGate abortPipeline: true
-            }
-        }
+// 	       stage('SonarQube Analysis') {
```

Comment:
Why not just delete?

Otherwise, add a TODO with comment as to why we are commenting out a lot of stuff (instead of removal).